dtfs-central-api/src/helpers/index.ts

```diff
@@ -1,11 +1,10 @@
 export { isNumber } from './isNumber';
-export { feeRecordCsvRowToSqlEntity } from './fee-record.helper';
+export { feeRecordCsvRowToSqlEntity } from './fee-record-helpers';
 export { executeWithSqlTransaction } from './execute-with-sql-transaction';
 export { calculateTotalCurrencyAndAmount } from './calculate-total-currency-and-amount';
 export { feeRecordsAndPaymentsMatch } from './fee-record-matching';
-export {
-  FeeRecordPaymentEntityGroup,
-  getFeeRecordPaymentEntityGroupsFromFeeRecordEntities,
-} from './get-fee-record-payment-entity-groups-from-fee-record-entities';
+export { getFeeRecordPaymentEntityGroups } from './get-fee-record-payment-entity-groups';
+export { getFeeRecordPaymentEntityGroupStatus } from './get-fee-record-payment-entity-group-status';
+export { getFeeRecordPaymentEntityGroupReconciliationData } from './get-fee-record-payment-entity-group-reconciliation-data';
 export { convertTimestampToDate } from './convert-timestamp-to-date';
-export { getLatestCompletedAmendmentCoverEndDate } from './tfm-facilities.helper';
+export { getLatestCompletedAmendmentCoverEndDate } from './tfm-facilities-helpers';
```

Comment:
What's the reason for adding `-helpers` to 2x of these helpers? I can see they had `.helper` before, but these 2x helper functions are the only helpers with "helper" in the name. I think we should just remove any instance of "helper" in these 2x file names.

Reply:
I think this was off the back of this comment you left: https://github.com/UK-Export-Finance/dtfs2/pull/3417#discussion_r1741729621 . I think perhaps wires got crossed with what the file names should be, just so I am clear do we want `file-name.helper.ts` for the main file but then just `file-name.test.ts` for the test file name?